src/field_theory/galois.lean

```diff
@@ -65,17 +62,11 @@ namespace is_galois
 instance self : is_galois F F :=
 ⟨is_separable_self F, normal_self F⟩
 
-@[priority 100] -- see Note [lower instance priority]
-instance to_is_separable [h : is_galois F E] : is_separable F E := h.1
-
-@[priority 100] -- see Note [lower instance priority]
-instance to_normal [h : is_galois F E] : normal F E := h.2
```

Comment:
There were three (3) ways to go from `is_galois F E` to `normal F E`, might as well fix this duplication.